sys/include/net/gnrc/ipv6.h

```diff
@@ -158,6 +158,8 @@ void gnrc_ipv6_demux(gnrc_netif_t *netif, gnrc_pktsnip_t *current,
  *
  *          This function may be used with e.g. a pointer to a (full) UDP datagram.
  *
+ * @pre Any GNRC_NETTYPE_IPV6 snip in pkt is contains a full IPv6 header.
```

Comment:
Just to be nice add `@ref GNRC_NETTYPE_IPV6`. Sadly, doxygen isn't smart enough to pick up macros and enum values itself to reference them.

Reply:
Fixed, thanks, good point.